MANIFEST

```diff
@@ -388,6 +388,9 @@ lib/LaTeXML/Package/babel.sty.ltxml
 lib/LaTeXML/Package/balance.sty.ltxml
 lib/LaTeXML/Package/bbm.sty.ltxml
 lib/LaTeXML/Package/bbold.sty.ltxml
+lib/LaTeXML/Package/beamer.cls.ltxml
+lib/LaTeXML/Package/beamer_support_preload.sty.ltxml
+lib/LaTeXML/Package/beamer_support_postload.sty.ltxml
```

Comment:
Although `beamer.cls.ltxml` would be a bit on the large side, it would be better to combine these three into a single binding.

Reply:
The original intent was to allow future compatibility for `beamerarticle.cls` (and share the helper code), but maybe I am thinking too general.